### ISSUE TITLE
Fix compatibility for 1.1.95

### ISSUE THREAD

**Issue by xerta555** [spicetify/spicetify-cli#1957]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.95.889.gfc9a996d-875
- Spicetify version: 2.13.1
```

### 📝 Description

Error when doing `spicetify backup apply enable-devtools`:
![image](https://user-images.githubusercontent.com/20888017/192612210-f7e7c7f2-15a0-4789-adef-bf5705854d1c.png)

Thanks in advance for fixing it.

**Comment by iamririxi** [comment]
It's already fixed #1955. Wait for release containing this fix

**Comment by xerta555** [comment]
> It's already fixed by #1955. Wait for release containing this fix

Ah yeah sorry...completly forgot that this pr fix this version.
